### ISSUE TITLE
CMC for small custom dataset

### ISSUE THREAD

**Issue by HaoliangZhou** [HobbitLong/CMC#72]
Thanks for open-sourcing your work, I have been trying to use CMC on my custom toy dataset which has 2 views (RGB, Depth). The number of RGB and Depth images is 4000. I have 2 dataloader, one for RGB data and another for depth. And use corresponding RGB and depth to be positive pair.
When I use NCECriterion, the reslut is like,
![VYKQ(~ZZMM}IC)9RSH3T`QS](https://user-images.githubusercontent.com/89777862/175956615-204939dc-e6ec-4008-af62-418b987786e5.png)
If use Softmax-CE, the result becomes,
![image](https://user-images.githubusercontent.com/89777862/175956766-60a02273-ca06-4c2b-b8a1-3e546cfbaf12.png)

Could you please give me some suggestion about train with dual-modal datasets or small datasets ?

Thanks a lot!
